ab_tool/static/ab_tool/js/experiment.js

```diff
@@ -50,6 +50,40 @@ angular.module('ABToolExperiment', []).controller(
         return Math.round(100 / $scope.experiment.tracks.length);
     }
     
+    $scope.percentsTotal100 = function() {
+        /**
+         * Returns a boolean as to whether or not the tracks add up to 100.
+         * If the return is false, raises an alert.
+         */
+        if ($scope.experiment.uniformRandom) {
+            // If uniformRandom is true, the actual track weights are ignored.
+            return true;
+        }
+        var i, len, sum;
+        for (i = 0, sum = 0, len = $scope.experiment.tracks.length; i < len; i++) {
+            var weighting = $scope.experiment.tracks[i].weighting;
```

Comment:
This loop makes sure that all tracks add up to 100, but it seems like a track can equal 0. am I right? If I am, do we want to allow tracks to have a weight of 0?